cmd/web3/main.go

```diff
@@ -229,6 +229,68 @@ func main() {
 				IncreaseGas(ctx, privateKey, network, c.String("tx"), c.String("amount"))
 			},
 		},
+		{
+			Name:  "replace",
+			Usage: "Replace transaction. If a transaction is still pending, you can attempt to replace it.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "private-key, pk",
+					Usage:       "The private key",
+					EnvVar:      pkVarName,
+					Destination: &privateKey,
+					Required:    true},
+				cli.StringFlag{
+					Name:     "nonce",
+					Usage:    "The nonce to replace.",
+					Required: true},
+				cli.StringFlag{
+					Name:     "to",
+					Usage:    "to address",
+					Required: true,
+				},
+				cli.StringFlag{
+					Name:     "amountd", // adding a d for backwards compatibility. If d, then it's decimal, otherwise, the old stuff.
+					Usage:    "The amount of GO or ETH in decimal format",
+					Required: true,
+				},
+				cli.Uint64Flag{
+					Name:     "gas-limit",
+					Usage:    "Gas limit (multiplied by price for total gas)",
+					Required: true,
+				},
+				cli.StringFlag{
+					Name:  "gas-price",
+					Usage: "Gas price to use, if left blank, will use suggested gas price.",
+				},
+			},
+			Action: func(c *cli.Context) {
+				toS := c.String("to")
+				if toS == "" {
+					fatalExit(errors.New("to address not set"))
+				}
+				var ok bool
+				// a := c.String("amount")
+				// amount, ok := new(big.Int).SetString(a, 10)
+				// if !ok {
+				// 	fatalExit(fmt.Errorf("invalid amount %v", a))
+				// }
+				amountd, err := decimal.NewFromString(c.String("amountd"))
+				if err != nil {
+					fatalExit(err)
+				}
+				amount := web3.DecToInt(amountd, 18)
```

Comment:
We added `ParseGwei/ParseBase/parseUnit` recently for decimal parsing. We should probably keep everything aligned one way or the other. Using the `decimal` package seems better than my custom code, but I wonder if the silent rounding will be surprising UX.